docs/src/main/sphinx/connector/teradata.md

```diff
@@ -0,0 +1,334 @@
+
+* - JSON
+  - JSON
+
+No other types are supported.
+
+
+## Querying Teradata
+The Teradata connector provides a schema for every Teradata database. You can see the available Teradata databases by running SHOW SCHEMAS:
+```
+SHOW SCHEMAS FROM teradata;
+```
+If you have a Teradata database named sales, you can view the tables in this database by running SHOW TABLES:
+```
+SHOW TABLES FROM teradata.sales;
+```
+You can see a list of the columns in the orders table in the sales database using either of the following:
+```
+DESCRIBE teradata.sales.orders;
+SHOW COLUMNS FROM teradata.sales.orders;
+```
+Finally, you can access the orders table in the sales database:
+```
+SELECT * FROM teradata.sales.orders;
+```
+## SQL support
+The connector provides read access to data and metadata in the Teradata database. In addition to the [globally available](https://trino.io/docs/current/language/sql-support.html#globally-available-statements) and [read operation](https://trino.io/docs/current/language/sql-support.html#read-operations) statements, the connector supports the following features:
+
+## Procedures
+The connector supports the following procedures:
+
+
+system.flush_metadata_cache()
+Flush the metadata cache for all schemas or a specified schema.
+
+Example usage:
+```
+CALL teradata.system.flush_metadata_cache();
+CALL teradata.system.flush_metadata_cache(schema_name => 'sales');
+```
+## Table functions
+The connector provides specific [table functions](https://trino.io/docs/current/functions/table.html) to access Teradata.
+
+
+query(varchar) -> table
+The query function allows you to query the underlying database directly. It requires syntax native to Teradata, because the full query is pushed down and processed in Teradata. This can be useful for accessing native features which are not available in Trino or for improving query performance in situations where running a query natively may be faster.
+
+For example, query the teradata catalog and use Teradata-specific functions:
+```
+SELECT
+  *
+FROM
+  TABLE(
+    teradata.system.query(
+      query => 'SELECT
+        employee_id,
+        RANK() OVER (ORDER BY salary DESC) as salary_rank
+      FROM
+        hr.employees
+      ORDER BY salary_rank'
+    )
+  );
+```
+## Performance
+The connector includes a number of performance improvements, detailed in the following sections.
+
+
+### Table statistics
+The Teradata connector can use [table and column statistics](https://trino.io/docs/current/optimizer/statistics.html) for [cost based optimizations](https://trino.io/docs/current/optimizer/cost-based-optimizations.html), to improve query processing performance based on the actual data in the data source.
+The statistics are collected by Teradata and retrieved by the connector. The table and column statistics are based on Teradata's Data Dictionary views.
+
+You can update statistics in Teradata by running:
+```
+COLLECT STATISTICS ON table_name;
+```
+### Pushdown
```

Comment:
https://github.com/Teradata/trino/blob/81808851a8d249bde9c5c50282895d2928cea38a/docs/src/main/sphinx/connector/postgresql.md?plain=1#L491 Please refer this code to add links